CNV/somatic/crsp_and_wgs/wdl/cnv_validation.wdl

```diff
@@ -0,0 +1,160 @@
+import "cnv_somatic_pair_workflow.wdl" as CNVSomaticPairWorkflow
+
+workflow CNVValidation {
+
+    ### CNV parameters
+    File intervals
+    File common_sites
+    File tumor_bam
+    File tumor_bam_idx
+    File? normal_bam
+    File? normal_bam_idx
+    File ref_fasta
+    File ref_fasta_dict
+    File ref_fasta_fai
+    File read_count_pon
+    String gatk_docker
+
+    File? gatk4_jar_override
+    Int? bin_length
+
+    File? gatk4_jar_override_evaluation
```

Comment:
@asmirnov239 
Yes, this was so that I could use the GATK tool `CombineSegmentBreakpoints` from a different branch than I was using to simply run the CNV tools. 

Is there any action you would like me to take here?